ethereumetl/service/token_transfer_v2_extractor.py

```diff
@@ -69,9 +69,8 @@ def extract_transfer_from_log(self, receipt_log):
                 token_type="ERC1155")]
         
         elif topics[0] == ERC1155_TRANSFER_BATCH_TOPIC:
-            #todo cleanup
-            if len(topics_with_data) <= 10:
-                logger.warning("The number of topics and data parts is not equal to or greater than 10 in log {} of transaction {}"
+            if len(topics_with_data) < 6:
```

Comment:
why did this change to 6 instead of 10?

Reply:
Chatting with @andschneider on this but based on the [event signature](https://docs.openzeppelin.com/contracts/3.x/api/token/erc1155#IERC1155-TransferBatch-address-address-address-uint256---uint256--- ),  dont see why it should be 10

Reply:
`test_extract_transfer_from_receipt_log_erc1155_batch_with_single_token_id` fails without this change